lib/testing.dart

```diff
@@ -22,4 +22,8 @@
 ///           200,
 ///           headers: {'content-type': 'application/json'});
 ///     });
+library http.testing;
```

Comment:
Why do we need this?

Reply:
Something to hang the comment off. Minimizes the chance dartdoc gets confused and thinks one is annotating the import directive